### PR TITLE
fix: kernel tests ci

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/reusable_kernel_tests.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
           run_id: "id-${{ github.run_id }}"
           output_dir: "~/ansible_output_${{ github.run_id }}"
           repos:
-            libs: {name: "falcosecurity-libs", repo: "https://github.com/${{ github.event.inputs.libsrepo }}.git", version: "${{ github.event.inputs.libsversion }}"}
+            libs: {name: "falcosecurity-libs", repo: "https://github.com/${{ inputs.libsrepo }}.git", version: "${{ inputs.libsversion }}"}
```
Comment (FedeDP) [inline .github/workflows/reusable_kernel_tests.yaml:54]: Tested here: https://github.com/falcosecurity/libs/actions/runs/6505305275/job/17668682455
Now they are correctly expanded:
```
  libs: {name: "falcosecurity-libs", repo: "https://github.com/falcosecurity/libs.git", version: "master"}
```

Comment (FedeDP) [reply .github/workflows/reusable_kernel_tests.yaml:54]: The other syntax was the old syntax to access `workflow_dispatch` inputs, that was not working instead with `workflow_call`.
See https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/ for reference.

**Review by leogr — APPROVED** [review]

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**